aspnetcore/fundamentals/startup.md

```diff
@@ -1,75 +1,117 @@
 
 [!code-csharp[Main](../common/samples/WebApplication1/Startup.cs?highlight=4,7,11&start=40&end=55)]
 
-Adding services to the services container makes them available within your application via [dependency injection](xref:fundamentals/dependency-injection).
+## Services available in Startup
 
-## Services Available in Startup
+The web host provides some services that are available to the `Startup` class constructor. The app adds additional services via `ConfigureServices`. Both the host and app services are then available in `Configure` and throughout the application.
 
-ASP.NET Core dependency injection provides services during an application's startup. You can request these services by including the appropriate interface as a parameter on your `Startup` class's constructor or its `Configure` method. The `ConfigureServices` method only takes an `IServiceCollection` parameter (but any registered service can be retrieved from this collection, so additional parameters are not necessary).
+Some of the services typically requested by `Startup` methods:
 
-Below are some of the services typically requested by `Startup` methods:
+* In the constructor: `IHostingEnvironment`, `ILogger<Startup>`
+* In `ConfigureServices`: `IServiceCollection`
+* In `Configure`: `IApplicationBuilder`, `IHostingEnvironment`, `ILoggerFactory`
 
-* In the constructor:  `IHostingEnvironment`, `ILogger<Startup>`
-* In `ConfigureServices`:  `IServiceCollection`
-* In `Configure`:  `IApplicationBuilder`, `IHostingEnvironment`, `ILoggerFactory`
+## The Configure method
 
-Any services added by the ``WebHostBuilder`` ``ConfigureServices`` method may be requested by the ``Startup`` class constructor or its ``Configure`` method. Use `WebHostBuilder` to provide any services you need during `Startup` methods.
+The [Configure](/dotnet/api/microsoft.aspnetcore.hosting.startupbase.configure) method is used to specify how the app responds to HTTP requests. The request pipeline is configured by adding [middleware](xref:fundamentals/middleware) components to an [IApplicationBuilder](/dotnet/api/microsoft.aspnetcore.builder.iapplicationbuilder) instance that's provided by [dependency injection](xref:fundamentals/dependency-injection). `IApplicationBuilder` is available to the `Configure` method, but it isn't registered in the service container. Hosting creates an `IApplicationBuilder` and passes it directly to `Configure` ([reference source](https://github.com/aspnet/Hosting/blob/release/2.0.0/src/Microsoft.AspNetCore.Hosting/Internal/WebHost.cs#L179-L192)).
 
-## The Configure method
+A typical web app configures the pipeline with support for a developer exception page, [BrowserLink](http://vswebessentials.com/features/browserlink), error pages, static files, and ASP.NET MVC:
```

Comment:
I don't think browser link is typical. The templates do this tho.

Reply:
I assume that's why Steve put that there. I left it there exactly for that reason: templates do it, and they prob will keep on doing it.

Reply:
In lieu of typical can you say the templates configure ...?

Reply:
Instead of "typical web app," I'll say "the ASP.NET Core templates."